### PR TITLE
Darkside helpers

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: zingolib/src/grpc_connector.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ impl GrpcConnector {
         Self { uri }
     }
 
-    pub(crate) fn get_client(
+    pub fn get_client(
```
Comment (pacu) [inline zingolib/src/grpc_connector.rs:41]: does this change in public interface need a CHANGELOG entry of some sort or additional docs?

File: darkside-tests/src/utils.rs [hunk 2]
```diff
@@ -384,3 +388,151 @@ impl TreeState {
...
+            height: target_height as u64,
+            ..tree_state
+        })
+        .await
+        .unwrap();
+    connector.apply_staged(target_height).await.unwrap();
+    target_height
+}
+
+/// Stage a block and transaction, then update tree state.
```
Comment (pacu) [inline darkside-tests/src/utils.rs:451]: is there a scenario in which updating the tree state is not desirable? If so this could be parameterized

**Review by pacu — COMMENTED** [review]
Did a generic pass on this PR. waiting for more context

File: darkside-tests/src/utils.rs (hunk 2 above)
Comment (zancas) [reply darkside-tests/src/utils.rs:451]: Good non-blocking idea.

**Review by zancas — APPROVED** [review]

**Review by zancas — APPROVED** [review]

**Pull request merged**